lite-locals.tf

```diff
@@ -0,0 +1,525 @@
+output "thor_max_jobs" {
+   value = var.thor_max_jobs
+}
+output "thor_num_workers" {
+   value = var.thor_num_workers
+}
+output "thor_node_size" {
+   value = local.aks_node_sizes.thor
+}
+output "thor_ns_spec" {
+   value = local.ns_spec[local.aks_node_sizes.thor]
+}
+output "thor_worker_cpus" {
+   value = local.thor_worker_cpus
+}
+output "thorWorkersPerNode" {
+   value = "local.ns_spec[${local.aks_node_sizes.thor}].cpu / local.thor_worker_cpus = ${local.thorWorkersPerNode}"
+}
+output "thor_worker_ram" {
+   value = "local.ns_spec[${local.aks_node_sizes.thor}].ram / local.thorWorkersPerNode = ${local.thor_worker_ram}"
+}
+output "nodesPer1Job" {
+   value = "var.thor_num_workers /  local.thorWorkersPerNode = ${local.nodesPer1Job}"
+}
+output "thorpool_max_capacity" {
+   value = "local.nodesPer1Job * var.thor_max_jobs = ${local.thorpool_max_capacity}"
+}
+locals {
+  thor_worker_cpus = 2
+
+  aks_node_sizes = {
+    roxie       = var.aks_roxie_node_size
+    serv        = var.aks_serv_node_size
+    spray       = var.aks_spray_node_size
+    thor        = var.aks_thor_node_size
+  }
+
+  ns_spec = {
+    "large" = {
+       cpu = 2
+       ram = 8
+    }
+    "xlarge" = {
+       cpu = 4
+       ram = 16
+    }
+    "2xlarge" = {
+       cpu = 8
+       ram = 32
+    }
+    "4xlarge" = {
+       cpu = 16
+       ram = 64
+    }
+  }
+
+  twpn = "${ local.ns_spec[local.aks_node_sizes.thor].cpu / local.thor_worker_cpus }"
+  thorWorkersPerNode = ceil(local.twpn) == local.twpn? local.twpn : "local.thorWorkersPerNode, ${local.twpn}, is not an integer because local.ns_spec[${local.aks_node_sizes.thor}].cpu, ${local.ns_spec[local.aks_node_sizes.thor].cpu}, is not a multiple of local.thor_worker_cpus, ${local.thor_worker_cpus}."
+
+  twr = "${local.ns_spec[local.aks_node_sizes.thor].ram / local.thorWorkersPerNode }"
+  thor_worker_ram = ceil(local.twr) == local.twr? local.twr : "local.thor_worker_ram, ${local.twr}, is not an integer because local.ns_spec[${local.aks_node_sizes.thor}].ram, ${local.ns_spec[local.aks_node_sizes.thor].ram}, is not a multiple of local.thorWorkersPerNode, ${local.thorWorkersPerNode}."
+
+  np1j = "${var.thor_num_workers /  local.thorWorkersPerNode }"
+  nodesPer1Job = ceil(local.np1j) == local.np1j? local.np1j : "local.nodesPer1Job, ${local.np1j}, is not an integer because var.thor_num_workers, ${var.thor_num_workers}, is not a multiple of local.thorWorkersPerNode, ${local.thorWorkersPerNode}."
+
+  thorpool_max_capacity = ceil("${ local.nodesPer1Job * var.thor_max_jobs }")
```

Comment:
This does not work in all cases (one Thor worker, one max job); nodesPer1Job becomes 0.5 and fails validation.

```
│ Error: Invalid operand
│ 
│   on lite-locals.tf line 66, in locals:
│   66:   thorpool_max_capacity = ceil("${ local.nodesPer1Job * var.thor_max_jobs }")
│     ├────────────────
│     │ local.nodesPer1Job is "local.nodesPer1Job, 0.5, is not an integer because var.thor_num_workers, 1, is not a multiple of local.thorWorkersPerNode, 2."
│ 
│ Unsuitable value for left operand: a number is required.
```